python/tvm/relay/build_module.py

```diff
@@ -555,6 +559,9 @@ def create_executor(kind="debug", mod=None, device=None, target="llvm"):
     else:
         device = _nd.device(str(target), 0)
 
+    if params is not None:
+        mod = IRModule.from_expr(bind_params_by_name(mod["main"], params))
```

Comment:
Do we need to consider what if the mod does not contain a "main" function, or the mod contains multiple functions(subgraphs), each with a different params dict?

Reply:
I think it's fine to throw an error if cannot find "main" in the Module, so resolve this comment.

Reply:
Yes, and it is passing all the CI stages.